buildSrc/src/main/kotlin/LibsConfig.kt

```diff
@@ -20,14 +31,14 @@ fun Project.applyLibrariesConfiguration() {
 
     configurations {
         create("shade")
-        getByName("archives").extendsFrom(getByName("default"))
     }
 
     group = "${rootProject.group}.worldedit-libs"
 
     val relocations = mapOf(
-            "net.kyori.text" to "com.sk89q.worldedit.util.formatting.text",
-            "net.kyori.minecraft" to "com.sk89q.worldedit.util.kyori"
+        "net.kyori.text" to "com.sk89q.worldedit.util.formatting.text",
+        "net.kyori.minecraft" to "com.sk89q.worldedit.util.kyori",
+        "net.kyori.adventure.nbt" to "com.sk89q.worldedit.util.nbt"
```

Comment:
This will need to be hold off until https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/918 is merged.

Reply:
I'll revert this.